Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/GridCollection.cs

```diff
@@ -0,0 +1,451 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    /// <summary>
+    /// An Object Collection is simply a set of child objects organized with some
+    /// layout parameters.  The object collection can be used to quickly create 
+    /// control panels or sets of prefab/objects.
+    /// </summary>
+    public class GridCollection : BaseCollection
+    {
+        #region public members
+
+        /// <summary>
+        /// Type of surface to map the collection to.
+        /// </summary>
+        [Tooltip("Type of surface to map the collection to")]
+        public SurfaceTypeEnum SurfaceType = SurfaceTypeEnum.Plane;
```

Comment:
I'm pretty sure some of these enums are already defined in core.

Reply:
I didn't see any existing enums when I did a search -> related to the previous comment. Which should we pull in?

Reply:
No worries, I just wanted to double check. I couldn't find any either. The only one that looks similar is PivotAxis, but I think these area all good here.